.node-version

```diff
@@ -0,0 +1 @@
+6
```

Comment:
I'm testing against [several node versions](https://github.com/lob/pdffonts/pull/1/files#diff-354f30a63fb0907d4ad57269548329e3R16), which is why the test code isn't ES6, but I thought it would be better to make it use the latest version supported. I can change it to `5` but since I'm testing against `6` also, I didn't think it made much sense.